recipes/seqsero2s/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "SeqSero2S" %}
+{% set version = "1.1.3" %}
+{% set release = "c4e0712a6ed672a4908c61b62d63873d51831d1d" %}
+{% set sha256 = "a3bf53993e687323b75f99fdeda2873da79346957ef25c9cb56daeda7fb67eeb" %}
+
+package:
+  name: '{{ name | lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/crashfrog/SeqSero2S/archive/refs/tags/{{ version }}.zip
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
+  run_exports:
+    - {{ pin_subpackage('seqsero2s', max_pin="x.x") }}
+
+requirements:
+  host:
+    - python >=3
+    - pip
```

```suggestion
    - pip
    - setuptools
```
Adding setuptools to the `host` section should fix the error.